Cargo.toml

```diff
@@ -135,3 +135,7 @@ sqlx = { git = "https://github.com/launchbadge/sqlx.git", rev = "42dd78fe931df65
 # [patch.'https://github.com/helium/proto']
 # helium-proto = { path = "../../proto" }
 # beacon = { path = "../../proto" }
+
+[patch.'https://github.com/helium/proto']
+helium-proto = { git = "https://www.github.com/helium/proto.git", branch = "jg/disco-shares-v2" }
```

Comment:
Might be nice to leave this as a comment when we move back to main branch. It probably won't be the last time we need to override a proto branch of a dep.